lm_eval/models/huggingface.py

```diff
@@ -494,23 +507,6 @@ class AutoCausalLM(HuggingFaceAutoLM):
     AUTO_MODEL_CLASS = transformers.AutoModelForCausalLM
     AUTO_PEFT_CLASS = peft.PeftModel
 
-    def _create_auto_tokenizer(
-        self,
-        *,
-        pretrained: str,
-        revision: str,
-        subfolder: str,
-        tokenizer: Optional[str] = None,
-    ) -> transformers.PreTrainedTokenizer:
-        tokenizer = super()._create_auto_tokenizer(
-            pretrained=pretrained,
-            revision=revision,
-            subfolder=subfolder,
-            tokenizer=tokenizer,
-        )
-        tokenizer.padding_side = "left"
-        return tokenizer
-
```

Comment:
Looks like it was an artifact of the last version update. It's an old code with a different function signature compared to the base class (which cost me a lot of head-scratching...)